appinfo/info.xml

```diff
@@ -2,10 +2,18 @@
 <info>
     <id>contacts</id>
     <name>Contacts</name>
-    <summary>A contacts app for ownCloud</summary>
+    <summary>Manage your CardDAV contacts from a feature-rich web-interface</summary>
     <description>
-The ownCloud contacts app is a user interface for ownCloud's CardDAV server.
-Easily sync contacts from various devices with your ownCloud and edit them online.</description>
+Contacts is a user interface for ownCloud's CardDAV server. ownCloud Server comes with a CardDAV component that allows users to synchronize and backup their address books with ownCloud and across multiple devices.
+Thanks to this feature users are provided with an alternative to proprietary contact synchronization services giving them back control over their data once more.
+
+The Contacts application is an extension to ownCloud's CardDAV interface. It equips ownCloud with a feature-rich frontend for the CardDAV server component, supports multiple address books and empowers users to easily manage their contacts from a web interface for ownCloud.
+Users may find the URL for device synchronization (to configure synchronization in iOS, Android or other CardDAV-capable applications) in the 'Settings' section on the bottom left inside the application (link symbol).
+Furthermore users can share address books with other users or groups (read-only/editable) enabling to synchronize address books not only across devices but also across users.
+In addition to managing synchronized contacts the application also offers a feature to import contacts from vCards (.vcf).
+
+### Resources
+- [ownCloud Contacts & Calendar Documentation](https://doc.owncloud.org/server/10.0/user_manual/pim/index.html)</description>
```

Comment:
We agreed to add docs to the description, wtf? Are those tags read and displayed by the marketplace?